ImmuDB4Net/ImmuClient.cs

```diff
@@ -520,14 +715,26 @@ public async Task<Entry> VerifiedGetAtTx(byte[] key, ulong tx)
             AtTx = tx
         };
 
-        return await VerifiedGet(keyReq, State());
+        return await VerifiedGet(keyReq, State);
     }
 
+    /// <summary>
+    /// Retrieves with authenticity check the value for a specific key since a transaction ID
```

Comment:
The meaning of `SinceTx` is somewhat unintuitive. It does not mean to get the change that happened after that transaction but it means that we require the server to index at least that transaction. This is useful when working with asynchronous index processing (noWait=true sent for GRPC mesages).

Thus I'd rephrase the summary to:
```suggestion
    /// Retrieves with authenticity check the value for a specific key. Changes to the key after given transaction can be ignored.
```

Reply:
ok